addons/block_code/ui/picker/categories/category_factory.gd

```diff
@@ -404,6 +404,46 @@ static func get_general_blocks() -> Array[Block]:
 	b.category = "Math"
 	block_list.append(b)
 
+	b = BLOCKS["parameter_block"].instantiate()
+	b.block_name = "randf"
+	b.variant_type = TYPE_FLOAT
+	b.block_format = "Random number between 0.0 and 1.0"
+	b.statement = "randf()"
+	b.defaults = {}
+	b.category = "Math"
+	b.tooltip_text = "Generate a random floating point number between 0.0 and 1.0 inclusively ([0.0, 1.0])"
+	block_list.append(b)
+
+	b = BLOCKS["parameter_block"].instantiate()
+	b.block_name = "randf_range"
+	b.variant_type = TYPE_FLOAT
+	b.block_format = "Random number between {from: FLOAT} and {to: FLOAT}"
+	b.statement = "(randf_range({from}, {to}))"
+	b.defaults = {"from": -1.0, "to": 1.0}
+	b.category = "Math"
+	b.tooltip_text = "Generate a random floating point number between [i]from[/i] and [i]to[/i] inclusively"
+	block_list.append(b)
+
+	b = BLOCKS["parameter_block"].instantiate()
+	b.block_name = "randi"
+	b.variant_type = TYPE_INT
+	b.block_format = "Random number between 0 and 4294967295"
+	b.statement = "randi()"
+	b.defaults = {}
+	b.category = "Math"
+	b.tooltip_text = "Generate a random unsigned 32-bits integer number between 0 and 2^32-1 inclusively ([0, 2^32-1])"
+	block_list.append(b)
+
+	b = BLOCKS["parameter_block"].instantiate()
+	b.block_name = "randi_range"
+	b.variant_type = TYPE_INT
+	b.block_format = "Random number between {from: INT} and {to: INT}"
+	b.statement = "(randi_range({from}, {to}))"
+	b.defaults = {"from": -1, "to": 1}
```

Comment:
Maybe the 2 "random integer" blocks can be one, if the defaults are `from: 0, to: 2^32-1`. Then the code (statement) can be:
```
randi() if {from} == 0 and {to} == 2^32-1 else randi_range({from}, {to})
```

Reply:
I will merge the two blocks, because they are pretty similar. Also I will do the same for randi.

I feel the if statement is unnecessary because after some thought, I believe that the randi method is not useful in our case. I don't see a case where you would want such a high number of random values. So I will just the let the users enter the range they desire.

Reply:
> I will merge the two blocks, because they are pretty similar. Also I will do the same for randi.
> 
> I feel the if statement is unnecessary because after some thought, I believe that the randi method is not useful in our case. I don't see a case where you would want such a high number of random values. So I will just the let the users enter the range they desire.

Yes, very well thought @jfcharron. Programming with blocks should be easy and providing multiple blocks that do the same can be confusing to the end users.